pike/model.py

```diff
@@ -1734,6 +1734,20 @@ def validate_negotiate_info(self, tree):
 
         return res
 
+    def query_network_interface_info(self, tree):
```

Comment:
can you split this into 2 functions? one that just returns the request, and another that does the transceive and returns the response.

(see other examples in pike/model.py)

Reply:
Was basing the logic from validate_negotiate_info. Fixed.